.env.development

```diff
@@ -1,11 +1,11 @@
-VITE_API_URL=http://localhost:3003/
+VITE_API_URL=https://horse.link/api
 VITE_PORT=3000
 VITE_GOERLI_URL=http://localhost:8545
 VITE_GOERLI_MNEMONIC=track evoke moment famous camera loud mirror action clip twelve deposit insect
 VITE_REST_FOR_MARKETS=true
 VITE_REST_FOR_VAULTS=true
 VITE_REST_FOR_TOKEN=true
 VITE_REST_FOR_BET=true
-VITE_ALCHEMY_API_KEY=<ALCHEMY_KEY>
+VITE_ALCHEMY_API_KEY=nj04KvcteO8qScoGLSYrz0p_tseWlb28
```

Comment:
i don't think we should commit the api key

Reply:
done.